### ISSUE TITLE
Is it possible to train Chinese data directly based on English models?

### ISSUE THREAD

**Issue by yfq512** [project-baize/baize-chatbot#21]
Is it possible to train Chinese data directly based on English models-[Baize-7B](https://huggingface.co/project-baize/baize-lora-7B) ？if not, why?

**Comment by JetRunner** [comment]
Of course, but it won't be as good as a Baize model trained on a Chinese foundation model.
